### PR TITLE
Fix Part of #2833 : Added missing named arguments in ImageRegionSelectionInteractionViewTest

### PR DESCRIPTION
**Pull request by Karanjot-singh**
## Explanation
 Fixes Part of #2833 : Added missing named arguments in ImageRegionSelectionInteractionViewTest

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Karanjot-singh** [comment]
PTAL @anandwana001

File: app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt [hunk 1]
```diff
@@ -116,14 +116,14 @@ class ImageRegionSelectionInteractionViewTest {
...
-        clickPoint(0.3f, 0.3f)
+        clickPoint(pointX = 0.3f, pointY = 0.3f)
       )
 
       verify(onClickableAreaClickedListener)
         .onClickableAreaTouched(
           capture(regionClickedEvent)
         )
-      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent("Region 3"))
+      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent(regionLabel = "Region 3")) // ktlint-disable max-line-length
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt]: Please don't use `// ktlint-disable max-line-length`. you can simply break this line into multiple lines. 

```suggestion
      assertThat(regionClickedEvent.value).isEqualTo(
        NamedRegionClickedEvent(
          regionLabel = "Region 3"
        )
      )
```

File: app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt [hunk 2]
```diff
@@ -160,7 +160,7 @@ class ImageRegionSelectionInteractionViewTest {
           regionClickedEvent
         )
       )
-      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent("Region 2"))
+      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent(regionLabel = "Region 2")) // ktlint-disable max-line-length
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt]:
```suggestion
      assertThat(regionClickedEvent.value).isEqualTo(
        NamedRegionClickedEvent(
          regionLabel = "Region 2"
        )
      )
```

File: app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt [hunk 3]
```diff
@@ -219,7 +219,7 @@ class ImageRegionSelectionInteractionViewTest {
...
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt]:
```suggestion
      assertThat(regionClickedEvent.value).isEqualTo(
        NamedRegionClickedEvent(
          regionLabel = "Region 2"
        )
      )
```

File: app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt [hunk 4]
```diff
@@ -243,7 +243,7 @@ class ImageRegionSelectionInteractionViewTest {
         .onClickableAreaTouched(
           capture(regionClickedEvent)
         )
-      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent("Region 3"))
+      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent(regionLabel = "Region 3")) // ktlint-disable max-line-length
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt]:
```suggestion
      assertThat(regionClickedEvent.value).isEqualTo(
        NamedRegionClickedEvent(
          regionLabel = "Region 3"
        )
      )
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt (hunk 1 above)
Comment (Karanjot-singh) [reply app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt]: Done!

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @Karanjot-singh

**Pull request merged**